src/pages/Models/Details/__snapshots__/ModelDetails.test.js.snap

```diff
@@ -4,7 +4,823 @@ exports[`ModelDetail Container renders the details pane 1`] = `
 <div
   className="model-details"
 >
-  <InfoPanel>
+  <InfoPanel
```

Comment:
I understand that this is just an update to an existing test but we had previously discussed the limited usefulness of snapshots that are this large. Would you mind updating the assertions in this branch to be more focused? Thanks!

Reply:
I think you marked this as resolved but didn't update the assertions :)